Description: This patch removes annoying (useless) message on stderr. This is a
vendor patch for eveyone that derives from Ubuntu.
Origin: https://bugs.edge.launchpad.net/ubuntu/+source/a52dec/+bug/124951/comments/1
Bug-Ubuntu: https://bugs.edge.launchpad.net/ubuntu/+source/a52dec/+bug/124951
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=441693
Forwarded: not-needed
Index: lucid/liba52/imdct.c
===================================================================
--- lucid.orig/liba52/imdct.c	2009-12-24 06:49:48.000000000 +0200
+++ lucid/liba52/imdct.c	2009-12-24 07:06:26.000000000 +0200
@@ -425,7 +425,7 @@
     } else
 #endif
     {
-	fprintf (stderr, "No accelerated IMDCT transform found\n");
+	// fprintf (stderr, "No accelerated IMDCT transform found\n");
 	ifft128 = ifft128_c;
 	ifft64 = ifft64_c;
     }
